config/filter.d/asterisk.conf

```diff
@@ -19,7 +19,7 @@ iso8601 = \d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d+[+-]\d{4}
 log_prefix= (?:NOTICE|SECURITY)%(__pid_re)s:?(?:\[C-[\da-f]*\])? \S+:\d*( in \w+:)?
 
 failregex = ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s Registration from '[^']*' failed for '<HOST>(:\d+)?' - (Wrong password|Username/auth name mismatch|No matching peer found|Not a local domain|Device does not match ACL|Peer is not supposed to register|ACL error \(permit/deny\)|Not a local domain)$
-            ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s Call from '[^']*' \(<HOST>:\d+\) to extension '\d+' rejected because extension not found in context 'default'\.$
+            ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s Call from '[^']*' \(<HOST>:\d+\) to extension '[^']*' rejected because extension not found in context
```

Comment:
because
0. it was anchored at the end before
1. more stringent expression is better/less vulnerable.  Sure thing longer one has higher chance of 'bugs' to creep in.  But with both ends anchored it becomes more robust to such misspecifications.  Minimizing the chance of false positives is our goal.
2. more stringent expression better captures our assumptions on the situation.  Possible extensions to the message might mean different things. again -- we need no false positives, we can allow for false negatives and adjust it later
3. performance might actually get better happen regexp matching engine starts matching from the end and immediately finds a string to reject